### PR TITLE
removed referring to a variable BASE_WWW

### PR DESCRIPTION
**Pull request by zagovorichev**

### CODE REVIEW

File: install/class.ExtensionInstaller.php [hunk 1]
```diff
@@ -133,12 +134,11 @@ public function applyAccessRules()
...
-            ClientLibRegistry::getRegistry()->register($this->extension->getId(),$extManifestConsts['BASE_WWW']. 'js');
-            ClientLibRegistry::getRegistry()->register($this->extension->getId().'Css',$extManifestConsts['BASE_WWW']. 'css');
-            
-        }
+    {
+        $jsPath = trim(Template::js('', $this->extension->getId()), '/');
+        ClientLibRegistry::getRegistry()->register($this->extension->getId(), $jsPath);
+
+        $cssPath = trim(Template::css('', $this->extension->getId()), '/');
+        ClientLibRegistry::getRegistry()->register($this->extension->getId().'Css', $cssPath);
```
Comment (antoinerobin) [inline install/class.ExtensionInstaller.php]: you should not have this 'css' here

File: scripts/update/Updater.php [hunk 2]
```diff
@@ -137,13 +138,8 @@ public function update($initialVersion) {
...
             foreach ($extensionManager->getInstalledExtensions() as $extension) {
-                $extManifestConsts = $extension->getConstants();
-                if (isset($extManifestConsts['BASE_WWW'])) {
-
-                    ClientLibRegistry::getRegistry()->register($extension->getId(), $extManifestConsts['BASE_WWW'] . 'js');
-                    ClientLibRegistry::getRegistry()->register($extension->getId() . 'Css', $extManifestConsts['BASE_WWW'] . 'css');
-
-                }
+                ClientLibRegistry::getRegistry()->register($extension->getId(), Template::js('', $extension->getId()));
+                ClientLibRegistry::getRegistry()->register($extension->getId().'Css', Template::css('', $extension->getId()));
```
Comment (antoinerobin) [inline scripts/update/Updater.php]: here neither

Comment (antoinerobin) [inline scripts/update/Updater.php]: Why don't you trim your js and css here ?

Comment (zagovorichev) [inline scripts/update/Updater.php]: because of unit tests, they failed when I did it here

File: install/class.ExtensionInstaller.php (hunk 1 above)
Comment (zagovorichev) [inline install/class.ExtensionInstaller.php]: Why do you think so? Could you explain me, please? I think that used by frontend, but if you see another new way, that not for this task
I mean: for me important to delete BASE_WWW usage, each another changes should be in other PR

Comment (antoinerobin) [inline install/class.ExtensionInstaller.php]: the css is already part of the $cssPath so you don't have to add it. Plus the case wasn't right

File: scripts/update/Updater.php (hunk 2 above)
Comment (antoinerobin) [inline scripts/update/Updater.php]: don't get why, it is just an update script it shouldn't impact unit test

File: install/class.ExtensionInstaller.php (hunk 1 above)
Comment (antoinerobin) [inline install/class.ExtensionInstaller.php]: I read it too fast, your right

File: scripts/update/Updater.php (hunk 2 above)
Comment (zagovorichev) [inline scripts/update/Updater.php]: because someone wrote test before me, and I don't want to broke it

**Pull request merged**